packages/view-types/spatial/src/SpatialSubscriber.js

```diff
@@ -492,6 +493,31 @@ export function SpatialSubscriber(props) {
     return imageLayerLoaders.map((ll, index) => (shouldUseFullData(ll, index) ? { ...ll, data: ll.data[0] } : ll));
   }, [imageLayerLoaders, useFullResolutionImage, meta]);
 
+  const getChannelLabels = useMemo(() => {
+    let channelNames = [];
+    let channelColors = [];
+
+    const segmentationLayerLoaders = obsSegmentations && obsSegmentationsType === 'bitmask' ? obsSegmentations.loaders : null;
+    if (cellsLayer && obsSegmentationsType === 'bitmask' && cellsLayer.length > 0) {
+      channelNames = cellsLayer.map(layer => segmentationLayerLoaders?.[layer.index])[0]?.channels;
+      channelColors = cellsLayer[0].channels.map(layer => layer.color) || [];
```

Comment:
When I tested with a few examples, this part of the code was never used. The reason I added it is that it is used in the `LayerControllerSubscriber` component (see around line 113). @keller-mark do you know in which cases this part of the code will be called? Is it safe to remove it in the function here?

Reply:
We would not want to display any channel names for segmentation layers so this code can be removed.